client/src/App.js

```diff
@@ -0,0 +1,27 @@
+import './App.css';
+import Topbar from './components/Topbar/Topbar';
+import CardNavegation from './components/CardNavegation/CardNavegation';
+import React from 'react';
```

Comment:
It isn't something I'll grade you down on, but it is worth your while to organize your imports. Start with any outside libraries and then internal references.

Reply:
I fixed the organization of the imports hopes it looks better now.